src/PluginClient.php

```diff
@@ -62,6 +62,36 @@ public function __construct($client, array $plugins = [], array $options = [])
     }
 
     /**
+     * Adds a plugin to the client.
+     *
+     * @param Plugin $plugin
+     */
+    public function addPlugin(Plugin $plugin)
```

Comment:
Sure, use `addPlugin`. But make sure to write a comment that the plugin added is appended to the end of the plugin queue